.github/workflows/ci.yml

```diff
@@ -277,6 +277,7 @@ jobs:
       IREE_CPU_DISABLE: 1
       IREE_VULKAN_DISABLE: 0
       IREE_CUDA_DISABLE: 0
+      IREE_HIP_DISABLE: 1
```

Comment:
We should have a CI job running with HIP enabled that includes the HAL CTS tests. Need more (reliable) AMDGPU runners for that though...

Reply:
+100! Though that's not the job for this pull request I guess. :D

Reply:
Non-experimental code should at least be built on CI. Looks like this is, so that's good.